modules/SocketManager/module/auto/SocketManager.yml

```diff
@@ -60,6 +60,9 @@ cdefs: &cdefs
 - SOCKETMANAGER_CONFIG_MAX_SOCKETS:
     doc: "Maximum number of sockets supported"
     default: 1024
+- SOCKETMANAGER_CONFIG_MAX_POLL_TIMEOUT:
+    doc: "Restrict the poll time out (in ms) to improve responsiveness. -1 for infinite"
```

Comment:
improve responsiveness is vague.
i think this bounds the max amount of time wait between requesting IND_SOC_RUN_STATUS_EXIT and returning from ind_soc_select_and_run

1024 is sort of odd for a ms value, since its 1.024 seconds... what about 1000?

Reply:
1000 works too. Yes I also wanted it to be able to perform the infinite poll if explicitly desired.